src/pages/NewStory.js

```diff
@@ -137,6 +137,7 @@ const NewStory = () => {
       setDescriptionError(true)
       return
     }
+    data.description = data.description.replace(/"/g, '\\"') // Replace all occurences of " with \"
```

Comment:
why are we doing this manually?

Reply:
When a link is added in the description text, it was giving errors upon submitting. Ckeditor does not seem to be doing it automatically. I am not sure if this is a bug with ckeditor.